src/script/interpreter.h

```diff
@@ -187,6 +209,11 @@ struct PrecomputedTransactionData
     explicit PrecomputedTransactionData(const T& tx);
 };
 
+/* Standard Template Hash Declarations */
```

Comment:
minor: I think all mention to “Standard” template can be removed and just `s/standard/default/g` or `s/standard/primary/g`. We have already `IsStandard` as a checker utility for policy sanitization, though here IIUC it’s meant to be “standard” as the default template hash 32-byte as told by BIP 119, which does not preclude future template that wouldn’t be necessarily the default 32-byte one.